meson.build

```diff
@@ -267,12 +267,13 @@ endif
 
 # tensorrt
 nvinfer_dep = dependency('', required: false)
-nvparsers_dep = dependency('', required: false)
+# nvparsers_dep = dependency('', required: false)
```

Comment:
please create nvonnxparser_dep empty dependency here for non-nvidia users. (and don't comment out removed variables)

Reply:
Done